src/amoc_controller.erl

```diff
@@ -288,6 +343,10 @@ node_userids(Start, End, Nodes, NodeId) when is_integer(Nodes), Nodes > 0,
 interarrival() ->
     amoc_config:get(interarrival, ?INTERARRIVAL_DEFAULT).
 
+-spec validation_interval() -> integer().
+validation_interval() ->
+    amoc_config:get(scenario_checking_interval, ?CHECKING_INTERVAL_DEFAULT).
+
```

Comment:
More consistent naming would be better - `validation_interval/0` returns config param `scenario_checking_interval`.